buildrelease.sh

```diff
@@ -57,8 +57,8 @@ do
   mv ../docs/output/assembled/$project ./docs
 done
 
-rm ./index.html
-mv ../docs/output/assembled/index.html ./
+rm ./docs/index.html
```

Comment:
We probably don't actually need this line at all, given that we're just going to put a file in place in the next line anyway. Just remove the line?

Reply:
I switched to ```mv -f``` instead of deleting the file.  This ensures we don't get prompted when we try and overwrite the file.